src/uharfbuzz/_harfbuzz.pyx

```diff
@@ -451,3 +468,7 @@ def ot_layout_table_get_script_tags(face: Face, tag: str) -> List[str]:
         packed = cstr
         tags.append(packed.decode())
     return tags
+
+
+cpdef void ot_font_set_funcs(Font font):
```

Comment:
Why cpdef here? I used def for all module level functions.

I don't mind cpdef (is it more efficient?) but we should use it consistently in the file then

Reply:
it's only more efficient in so far as you also need to call a cpdef function from within cython (in which case it uses its faster cdef version). But if we only call this from python, then def is ok.